sysbuild/sdp.cmake

```diff
@@ -2,16 +2,16 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
-# If it is enabled, include the eGPIO FLPR application in the build
-if(SB_CONFIG_EGPIO_FLPR_APPLICATION)
+# If it is enabled, include the SDP GPIO application in the build
+if(SB_CONFIG_SDP_GPIO)
   # Extract SoC name from related variables
   string(REPLACE "/" ";" split_board_qualifiers "${BOARD_QUALIFIERS}")
   list(GET split_board_qualifiers 1 target_soc)
   set(board_target_flpr "${BOARD}/${target_soc}/cpuflpr")
   set(target_soc)
 
   ExternalZephyrProject_Add(
-    APPLICATION flpr_egpio
+    APPLICATION sdp
```

Comment:
Changing it would clash with this requirement:
https://github.com/nrfconnect/sdk-nrf/pull/18179/files#diff-57dccb9cc9436d1b15eb307338c889c5a3a7eff7f6ef6929c401ed0096c57fc0R5

Does non-unique application name cause some issues?

Reply:
If there are no name conflicts then it's ok.